src/main/resources/META-INF/rewrite/jakarta-faces-4.yml

```diff
@@ -448,3 +449,77 @@ recipeList:
       oldFullyQualifiedTypeName: jakarta.faces.bean.ViewScoped
       newFullyQualifiedTypeName: jakarta.faces.view.ViewScoped
       ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.UpgradeFacesOpenSourceLibraries
+displayName: Upgrade Faces open source libraries
+description: Upgrade PrimeFaces, OmniFaces, and MyFaces libraries to Jakarta EE10 versions.
+recipeList:
+  - org.openrewrite.maven.ChangeDependencyClassifier:
+      groupId: org.primefaces
+      artifactId: primefaces
+      newClassifier: jakarta
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.primefaces
+      oldArtifactId: primefaces
+      newGroupId: org.primefaces
+      newArtifactId: primefaces
+      newVersion: 13.0.3
```

Comment:
Might make sense to [use Xrange version selectors here](https://docs.openrewrite.org/reference/dependency-version-selectors#xrange), such that we don't pin folks to any particular version, but for instance anything in `13.x` or `13.0.x`

Reply:
Great feedback updated them all to xrange!